### DISCUSSION TITLE
Follow a hashtag

### DISCUSSION THREAD

**Discussion by NCW-Srandall** [NickColley/semaphore#47] [New features and ideas]
When searching for a hashtag, I can interact with posts about the tag but not the tag itself. Some clients let me follow tags. Those tags do appear in my timelines in semaphore of course, which is great.
Also search results as a keyboard exercise work differently in that I can tab through options, but not use up/down as I can in other areas. Is this a deliberate design decision? is it just me who would have thought that arrowing through results is a similar workflow to timeline navigation? Do people search often enough that  it matters?

**Comment by NickColley** [comment]
I think there's some wider work to do around how the wider federated feeds and local feeds are consumed that could also include this, good idea thank you!
